hdf5-vol-daos.spec

```diff
@@ -197,8 +197,8 @@ done
 %endif
 
 %changelog
-* Mon Dec 14 2020 Kenneth Cain <kenneth.c.cain@intel.com> - 0.1-4.gb8e6afb18f
-- Update hdf5-vol to TBD and update virtual providess to build with libdaos.so.1
+* Mon Dec 14 2020 Kenneth Cain <kenneth.c.cain@intel.com> - 0.1-4.gfcbdc0b32d
```

Comment:
gfcbdc0b32d is not the hash that is generated for the RPM:

![image](https://user-images.githubusercontent.com/559317/104470774-14559b80-5588-11eb-960c-ccef0491d69c.png)

You can copy and paste the correct hash from my [previous suggestion](https://github.com/daos-stack/hdf5-vol-daos/pull/5#discussion_r553506992) if you wish.  It's correct there.

Since you need to fix the hash above, you should also update the date on this changelog entry.  It's quite a bit out of date as to when it will actually land.  A few days is never a problem but this one is a month out.

Reply:
Let me fix it up.
@mjean308 did you resolve this conversation? For some reason in the github UI when I look at the "Files changed" tab and see this comment, it shows up as resolved by mjean309. Based on @brianjmurrell comment on another conversation in this PRthe reviewer should be the one to resolve the issue after fixes are made.

Reply:
Out of curiosity so I can understand how to do this correctly in general, why do the number of characters in the commit hash vary across some of the entries in the %changelog in the specfile? I see 
0.1-1.gb324b90d4 (9 characters)
0.1-2.gb324b90d4 (9)
0.1-3.gb8e6afb18f (10)

the [suggestion](https://github.com/daos-stack/hdf5-vol-daos/pull/5#discussion_r553506992)  is to use  7 characters.

Reply:
> Out of curiosity so I can understand how to do this correctly in general, why do the number of characters in the commit hash vary across some of the entries in the %changelog in the specfile?

Because previous PRs incorrectly used hashes that were too long.

> the [suggestion](https://github.com/daos-stack/hdf5-vol-daos/pull/5#discussion_r553506992) is to use 7 characters.

That's the default for [git rev-parse --short](https://github.com/daos-stack/hdf5-vol-daos/blob/cfbc2154fad4109e15d1959d66615566a59adca2/Makefile#L8)

Reply:
> > Out of curiosity so I can understand how to do this correctly in general, why do the number of characters in the commit hash vary across some of the entries in the %changelog in the specfile?
> 
> Because previous PRs incorrectly used hashes that were too long.
> 
> > the [suggestion](https://github.com/daos-stack/hdf5-vol-daos/pull/5#discussion_r553506992) is to use 7 characters.
> 
> That's the default for [git rev-parse --short](https://github.com/daos-stack/hdf5-vol-daos/blob/cfbc2154fad4109e15d1959d66615566a59adca2/Makefile#L8)

Thanks. Yeah I saw the git rev-parse --short documentation, ut wasn't sure if somehow in the process of building an RPM if the tools involved in that might somehow generate package filenames that have different numbers of characters in different builds. That would be weird for sure, but was just checking.

The link to the suggestion is duly noted :)

Reply:
@kccain not sure why it appears I resolved the conversation.   I think I intended cancelled my comment about the date and then hit resolve instead...  ?     But I did not intend to resolve  it.